queries.sql

```diff
@@ -82,3 +82,275 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
   4 | Devimon | 2017-05-12    |               5 | t       |        11
 (2 rows)
 */
+
+
+/* /////////////////////////////////////////////////////////////////// */
+
+START TRANSACTION;
+
+UPDATE animals SET species = 'unspecified';
+/*
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg |   species
+----+------------+---------------+-----------------+---------+-----------+-------------
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | unspecified
+  2 | Gabumon    | 2018-11-15    |               2 | t       |         8 | unspecified
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 | unspecified
+  4 | Devimon    | 2017-05-12    |               5 | t       |        11 | unspecified
+  5 | Charmander | 2020-02-08    |               0 | f       |       -11 | unspecified
+  6 | Plantmon   | 2021-11-15    |               2 | t       |      -5.7 | unspecified
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 | unspecified
+  8 | Angemon    | 2005-06-12    |               1 | t       |       -45 | unspecified
+  9 | Boarmon    | 2005-06-07    |               7 | t       |      20.4 | unspecified
+ 10 | Blossom    | 1998-10-13    |               3 | t       |        17 | unspecified
+ 11 | Ditto      | 2022-05-14    |               4 | t       |        22 | unspecified
+(11 rows)
+*/
```

Comment:
### Optional
- [ ] Kindly remove the commented query results in this file, it is not needed and making the code hard to read